finish/src/main/java/io/openliberty/guides/system/SystemResource.java

```diff
@@ -31,10 +31,14 @@
   @GET
   @Produces(MediaType.APPLICATION_JSON)
   public Response getProperties() {
+    // tag::isInMaintenance[]
     if (!systemConfig.isInMaintenance()) {
       return Response.ok(System.getProperties()).build();
     } else {
+    // end::isInMaintenance[]
```

Comment:
- `isInMaintenance()` hotspot should not include `else` statement

Reply:
addressed.